RecoLocalTracker/SiPixelClusterizer/plugins/alpaka/PixelClustering.h

```diff
@@ -114,6 +114,14 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE::pixelClustering {
       return new_status;
     }
 
+    ALPAKA_FN_ACC
+    bool isMorphingModule(uint32_t moduleId, const uint32_t* morphingModules, uint32_t nMorphingModules) {
```

Comment:
if there are more than about 10 modules, it would be much more efficient to
  - sort the modules (once, on the cpu, in the constructor of the module)
  - use a binary search to check if a module is in the list

Reply:
Has this been addressed?